kolibri/core/tasks/api.py

```diff
@@ -84,6 +85,10 @@ def _job_to_response(self, job):
             "clearable": job.state in [State.FAILED, State.CANCELED, State.COMPLETED],
             "facility_id": job.facility_id,
             "extra_metadata": job.extra_metadata,
+            "scheduled_datetime": str(orm_job.scheduled_time),  # Output is UTC naive.
```

Comment:
This all looks fine - I just wonder if it might be better to call this `scheduled_time` to be consistent with the field?

Reply:
Chose `datetime` to reflect that `date` and `time` both are present together. `scheduled_datetime` felt more expressive. Am fine with what you say sir..?

Reply:
Sure, let's leave it as is. Someone consuming the API should not have to know about the ORM representation, so I think we're fine!

Reply:
Yeah, exactly!